draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -688,7 +687,7 @@ know whether the key will need to produce pure or pre-hashed signatures
 at key generation time. The External Mu mode described in {{externalmu}}
 avoids all of these operational concerns.
 
-The security reason for disallowing HashML-DSA is that the design of the
+Another minor reason for disallowing HashML-DSA is that the design of the
```

Comment:
"minor security reason"
Don't be afraid to call it "security". It is "immaterial" or "minor" for sure.

Reply:
Done.